src/pages/Settings/PaymentSettings.vue

```diff
@@ -131,9 +133,11 @@
 		<kv-lightbox
 			class="active-card-lightbox"
 			:visible="showActiveLightbox"
-			:title="`Unable to remove card ${lowerCaseDescription}`"
```

Comment:
Not a blocker, but another approach would be to allow HTML in the title prop in KvLightbox so we can add a span here instead of bypassing the standard title.

Reply:
That's certainly one approach. I also considered passing a prop with a css class for the title area...